doc: prepare new release 3.1.0 (#3561)
tracim/tracim: changelog.md

## 3.1.0 / 2020-09-14

### New Features

> - A notification wall is now available in header bar. You can now seen all new activities done in Tracim.

Reviewer: ```suggestion
- A notification wall is now available in header bar. You can now see the new activities done in Tracim in real time.
```
Just a suggestion, you don't need to apply if you prefer the other way, but as we do not received the activities made by the logged user, I tried to change the "all".

Author: Wording change. thanks for your proposal.

Reviewer 2: "in real time" seems good to keep, what do you think?

> - It is now possible to mention user directly with his Tracim username like this `@username`. All mention are visible in notification wall.

Reviewer: ```suggestion
- It is now possible to mention user directly with his Tracim username like this `@username`. All mentions are visible in notification wall.
```

Author: Wording change. thanks for your proposal.

> - A notification wall is now available in header bar. You can now seen all new activities done in Tracim.
> - It is now possible to mention user directly with his Tracim username like this `@username`. All mention are visible in notification wall.

Reviewer 2: ```suggestion
- A notification wall now displays the users' notification, who can see what's happening in Tracim at a glance. It can be opened using a button in the header bar.
- It is now possible to mention users directly using their Tracim username like this: `@username`. This matches the behavior of many widespread application chats. Mentions are visible in the notification wall.
```

Author: > displays the users' notification,

You says `the user's notification,` ?

Reviewer 2: It's all good. However, I added an 's' to notification.
